cluster/ci/monitoring/Makefile

```diff
@@ -5,6 +5,7 @@ prow-monitoring-deploy:
 	oc apply -f ./prometheus_expose.yaml
 	oc apply -f ./prometheus_operator_deploy.yaml
 	oc apply -f ./sinker_service_monitor.yaml
+	oc apply -f ./ghproxy_service_monitor.yaml
```

Comment:
No need of `ServiceMonitor` for ghproxy unless it has its own /metrics endpoint.
if it only push to pushgateway like tide, i have done it already.

Reply:
addressed